proto/jumpstarter/v1/jumpstarter.proto

```diff
@@ -40,13 +41,12 @@ service ControllerService {
   // Get information of the exporter of the given uuid
   rpc GetExporter(GetExporterRequest) returns (GetExporterResponse);
 
-  // Lease exporter
-  // Lease the exporter of the given uuid for a given amount of time
-  rpc LeaseExporter(LeaseExporterRequest) returns (LeaseExporterResponse);
-
-  // Release exporter
-  // Return the exporter of the given uuid early
-  rpc ReleaseExporter(ReleaseExporterRequest) returns (ReleaseExporterResponse);
+  // Get Lease
+  rpc GetLease(GetLeaseRequest) returns (GetLeaseResponse);
+  // Create Lease
+  rpc CreateLease(CreateLeaseRequest) returns (CreateLeaseResponse);
+  // Delete Lease
+  rpc DeleteLease(DeleteLeaseRequest) returns (DeleteLeaseResponse);
```

Comment:
May be we don't want to really delete the lease object (at least right away), for example, keeping a bit of lease history can help with visibility on the usage of the platform, who is using, and what is being used....

Would it make sense to use the RequestLease / ReleaseLease semantics?

Reply:
We can have https://github.com/kubearchive keeping the history for us.

Reply:
Since we need to delete Lease sooner or later, deleting them as a way of releasing lease sounds natural to me.

Reply:
Eventually kubearchive could make sense (for long term storage) but I would soft-delete for now, then we can delete over some age threshold (and whoever wants to use kubearchive can keep perpetual storage).

I am also ok with he Create/Delete semantics, as long as we keep the CR in kubernetes for a while.

Reply:
Let's name the method Request/Release, and leave delete or not an implementation detail.